source/voice_control/s3_box_voice_assistant.markdown

```diff
@@ -0,0 +1,85 @@
+- USB-C cable to connect the ESP32-S3-BOX-3
+
+{% include voice_assistant/add_wake_word_to_voice_assistant.md %}
+
+## Installing the software onto the ESP32-S3-BOX-3
+
+If you have used the ESP32-S3-BOX-3 on Home Assistant before and have it installed via ESPHome add-on, you first need to remove its configuration.
+
+{% include voice_assistant/install_esp_firmware_delete_firmware.md %}
+
+### To install the software on your ESP32-S3-BOX-3
+
+Before you can use this device with Home Assistant, you need to install a bit of software on it.
+
+1. While keeping this tutorial page open, open the [ESPHome projects page](https://esphome.io/projects/index.html) in a Chrome or Edge browser on a desktop. It does not work on a tablet or phone.
+   - In the **I want to create a** section, select **Voice assistant**.
+   - Select the image of the ESP32-S3-BOX-3.
+   - From the list, select the product variant that you have.
+   - Select the **Connect** button. If your browser does not support web serial, you will see a warning instead of a button.
+   - **For advanced users**: The configuration files are available on GitHub:
+     - [ESP32-S3-BOX](https://github.com/esphome/firmware/blob/main/voice-assistant/esp32-s3-box.yaml)
+     - [ESP32-S3-BOX-3](https://github.com/esphome/firmware/blob/main/voice-assistant/esp32-s3-box-3.yaml)
+     - [ESP32-S3-BOX-Lite](https://github.com/esphome/firmware/blob/main/voice-assistant/esp32-s3-box-lite.yaml)
+
+{% include voice_assistant/install_esp_firmware.md %}
+
+1.  Select the **ESPHome** integration. Under **Devices**, you should see the **ESP32-S3-BOX-3** listed.
+   ![ESP32-S3-BOX-3 discovered](/images/assist/m5stack-atom-echo-discovered-new-03.png)
+   - Your ESP32-S3-BOX-3 is connected to Home Assistant over Wi-Fi. You can now move it to any place in your home with a USB power supply.
+2.   Congratulations! You can now voice control Home Assistant via a ESP32 device with a display. Now give some commands.
+
+## Controlling Home Assistant
+
+1. Say your wake word. For this tutorial, use "OK, Nabu".
+2. Say a [supported voice command](/voice_control/builtin_sentences/). For example, *Turn on the light*.
+   - Once the intent has been processed, the LED lights up in green and Home Assistant confirms the action.
+      - Make sure you’re using the area name exactly as you defined it in Home Assistant.
+      - You can also ask a question, such as
+          - *Is the front door locked?*
+          - *Which lights are on in the living room?*
+3. Your command is not supported? Add your own commands using [a sentence trigger](/voice_control/custom_sentences/).
+
+## Turning off microphone or screen
+
+1. If you do not want to use a wake word for a while, you can turn off the microphone.
+   - Go to {% my integrations title="**Settings** > **Devices & Services**" %} and select the **ESPHome** integration.
+      - Under **ESP32-S3-BOX-3**, select **1 device**.
+      - Disable **Use wake word**.
```

Comment:
This button officially changed name this morning.
It is now called "Mute" 

![image](https://github.com/home-assistant/home-assistant.io/assets/5878296/acba9c8f-fb01-443d-9352-9d6dd2366a10)


Sorry about the last minute change